### DISCUSSION TITLE
can webpack update dist folder partily when build in production model ?

### DISCUSSION THREAD

**Discussion by qiudaoermu** [webpack/webpack#17340] [Ideas]
now webpack delete the old folder and creat new dist (if dist not exited, create it).
i want do this ,`npm run build` then, webpack will diff new dist file and old dist file, and only update the changed file or folder,

**Comment by alexander-akait** [comment]
It already works as you described if you use `output.clean: true`

**Reply by qiudaoermu** [reply]
it don not work~

**Reply by alexander-akait** [reply]
Without an example of configuration or problem I can't help sorry
